README.md

```diff
@@ -7,3 +7,7 @@ Unified Workflow Tools for use with UFS applications and beyond
 ## Documentation
 
 Comprehensive documentation is available for [the development version](https://uwtools.readthedocs.io/en/main/) and for [the latest release](https://uwtools.readthedocs.io/en/stable/).
+
+## Jupyter Notebook Examples of API Tools
+
+Jupyter notebook containing examples of how to utilize API access to uwtools is available <a href = "https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Fexample.ipynb" target="_blank" rel="noopener noreferrer">here</a>.
```

Comment:
I think you might just not have circled back to this ticket -- but did we settle on a both-README-and-RST-docs solution here, such that this bit might move to the RST docs?

Reply:
I support that move. Keep the binder button in the README, but keep descriptions of it in the RST docs.

Reply:
Yes I was going to submit a separate PR request for it just because it was editing a different file and I had it in a different branch, I am happy to add it to this PR if that is better!

Reply:
A separate PR is fine. I just wasn't sure whether we were waiting for changes to this PR before approving.

Reply:
PR 544 submitted to add a link to the User Guide